openfisca_uk/variables/person/taxes/income_tax.py

```diff
@@ -315,7 +317,7 @@ def formula(person, period, parameters):
             "pension_contributions",
         ]
         total_deductions = add(person, period, DEDUCTIBLE)
-        return total_deductions
+        return max_(0, total_deductions)
```

Comment:
interesting, do these deductions increase MTR heterogeneity?

Reply:
Yeah I think they do a lot - maybe we should add a variable for taxable income after deductions, and use that for the MTR plots?

Reply:
Actually, that probably erodes the meaning a bit though...